### DISCUSSION TITLE
New case for ESP32-S3

### DISCUSSION THREAD

**Discussion by airzl** [hoylabs/OpenDTU-OnBattery#1215] [General]
With the update going forward to an ESP32 S3 the old cases with the previous ESP is not fitting anymore. I am strugglig to find any reasonable teplate for new cases that can hold an ESP32-S3 together with NRF24L01+ with breakout board and a display.

Is there any advice what can be used or printed?

**Comment by Andre0be** [comment]
I am also looking for

**Reply by jobax1** [reply]
https://www.thingiverse.com/thing:6734089
